docs/ox_inventory/index.md

```diff
@@ -241,3 +241,11 @@ Make sure you download the `ox_inventory.zip` file as that one contains the bund
 
 If in case you wanted to edit the inventory UI you would have to build these files yourself.  
 To do so please read our [_Building inventory UI_](./Guides/inventory_ui.md) guide.
+
+### Stashes / trunks are not saved at server restart
+
+The only supported way of doing a restart is **txAdmin scheduled restart**.  
+txAdmin scheduled restart should be your only way of restarting a server in production (server with player in it).
```

Comment:
Remove `(server with player in it)`, at least I hope people know what production means.

Reply:
Actually nvm I can do it